SetupDataPkg/Tools/ConfigEditor.py

```diff
@@ -559,6 +562,9 @@ def __init__(self, master=None):
             Manufacturing_enabled = (char_ext2_data & (0x1 << 6)) >> 6
             print(f"Manufacturing : {Manufacturing_enabled:02X}")
 
+        self.bios_schema_xml_hash = BoardMiscInfo.get_schema_xml_hash_from_bios()
+        print(f"Schema XML Hash: {self.bios_schema_xml_hash}")

```

Comment:
Should the print be removed? It will show up on the serial console, not on the canvas, which makes it of limited use for the user to consume

Reply:
done